kickstarts/base.ks.erb

```diff
@@ -157,6 +157,8 @@ open-vm-tools
 
 %post --log=/root/anaconda-post.log
 
+yum -y install http://www6.atomicorp.com/channels/atomic/centos/7/x86_64/RPMS/wmi-1.3.14-4.el7.art.x86_64.rpm
```

Comment:
If you have the .el7.src.rpm for it, you can add it in https://github.com/ManageIQ/srpms, then I can kick off a build and have it available in COPR for us to use (our manageiq-scl for CentOS 7.1 repo already added/created in kickstart on line 19).